Session_1/transpotaion.js

```diff
@@ -0,0 +1,8 @@
+//Transportation on vacation 
+// http://www.codewars.com/kata/transportation-on-vacation
+
+
+function rentalCarCost(d) {
+  var total = d*40;
+  return d>=7 ? total-50 : (d>=3) ? total -20 : total;
```

Comment:
Update the name of the param `d` here so that it improves the readability. Also it would be better to group the code this way `(d>=3 ? total -20 : total)`